whatsdown/src/pages/homePage/components/ServiceCard.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from "react";
+import Box from "@mui/material/Box";
+import Card from "@mui/material/Card";
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
```

Comment:
group all these imports
```suggestion
import { Box, Card, CardActions, CardContent, Button, Typography } from "@mui/material";
```